Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5264,15 +5264,15 @@ MouseEventsSimulationEnabled:
 
 MutationEventsEnabled:
   type: bool
-  status: mature
+  status: internal
```

Comment:
I guess we should change this to `testable` for now.
Otherwise, we'd have to modify hundreds of layout tests, and that'll make the revert of this change harder.